Sources/Bits/ByteBuffer+peek.swift

```diff
@@ -11,20 +11,20 @@ extension ByteBuffer {
     }
 }
 
-// Peek Into String
+/// Peek Into String
 extension ByteBuffer {
     public func peekString(count: Int, skipping: Int = 0, encoding: String.Encoding) -> String? {
         guard readableBytes >= count + skipping else { return nil }
-        guard let bytes = getBytes(at: skipping, length: count) else { return nil }
+        guard let bytes = getBytes(at: readerIndex + skipping, length: count) else { return nil }
```

Comment:
D'oh. I should have caught this one. It also confused me at first that `readFoo` doesn't need the readerIndex but `getFoo` does... seems like NIO is missing a method in between that: gets at current index, but doesn't move the current index forward.

Reply:
cc @Lukasa

Reply:
I can see that, but I'm not 100% sure I agree there. It's definitely a bit of a question, but normally this isn't an issue because the `read` methods literally don't take an index at all: there's no `read(offsetBy:length:)` method. That said, I'd be open to being convinced that we do need them.